### PR TITLE
forwarding to writer

### PR DESCRIPTION
**Pull request by radutudor**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -37,44 +36,61 @@ func main() {
...
+				fw.NewForwarder(client, *writerUri),
+			),
+			newHealthService(config),
+		).listenAndServe(*port)
 	}
 	err := app.Run(os.Args)
 	if err != nil {
-		log.Errorf("App could not start, error=[%s]\n", err)
+		log.Errorf("App could not start, error=[%v]\n", err)
 		return
```
Comment (kovacshuni) [inline main.go]: better fatalf?

File: forwarder/Forwarder.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import (
+	"bytes"
+	"fmt"
+	tidUtil "github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type Forwarder struct {
```
Comment (bogdanguranda) [inline forwarder/Forwarder.go]: Do we need to expose this type if we have a constructor for it?

**Review by bogdanguranda — APPROVED** [review]

File: forwarder/Forwarder.go (hunk 2 above)
Comment (radutudor) [reply forwarder/Forwarder.go]: I don't really know how go works? Can I use (not create, use) an instance of Forwarder if it is not exposed?

Comment (radutudor) [reply forwarder/Forwarder.go]: done

**Review by kovacshuni — APPROVED** [review]

**Pull request merged**